packages/payload/src/queues/config/generateJobsJSONSchemas.ts

```diff
@@ -161,7 +161,7 @@ export function generateJobsJSONSchemas(
             return [workflow.slug, toReturn]
           }),
         ),
-        required: jobsConfig.tasks.map((task) => task.slug),
+        required: jobsConfig.workflows.map((workflow) => workflow.slug),
```

Comment:
I made workflows optional in this PR
https://github.com/payloadcms/payload/pull/9039

I'd like it to be possible that you can use the jobs queue with a configuration only having tasks, rather than being forced to use workflows.